tools/test/collect_cc_coverage.sh

```diff
@@ -132,6 +132,12 @@ function gcov_coverage() {
               mkdir -p "${COVERAGE_DIR}/$(dirname ${gcno_path})"
               cp "$ROOT/${gcno_path}" "${COVERAGE_DIR}/${gcno_path}"
           fi
+
+          # Extract gcov's version: the output of `gcov --version` contains the
+          # version as a set of major-minor-patch numbers, of which we extract
+          # the major version.
+          gcov_major_version=$("${GCOV}" --version | sed -n -E -e 's/^.*\s([0-9]+)\.[0-9]+\.[0-9]+\s?.*$/\1/p')
```

Comment:
Why are you using a different mechanism to extract the major version here?

Reply:
It's simply replacing what was removed in https://github.com/bazelbuild/bazel/pull/18878.

This approach handles using `llvm-cov gcov`  - the tests don't need to worry about that.

Reply:
That said, when I come to consolidate things, I'll probably use the same approach everywhere (no reason not to).

But right now, I don't want to change things everywhere in the tests...